README.md

```diff
@@ -171,6 +175,8 @@ spec:
       annotations:
         autocert.step.sm/name: hello-mtls-1h.default.svc.cluster.local
         autocert.step.sm/duration: 1h
+        autocert.step.sm/owner: "999:999"
```

Comment:
Change this example to user:group instead of uid:gid

Reply:
I will update the docs to explain uid/gid is required.